<launch>
  <arg name="marker_size" default="15.0"/>
  <arg name="max_new_marker_error" default="0.1"/>
  <arg name="max_track_error" default="1.0"/>
  <arg name="cam_image_topic" default="/zed2/point_cloud/cloud_registered"/>
  <arg name="cam_info_topic" default="/zed2/depth/camera_info"/>
  <arg name="output_frame" default="base_footprint"/>
  <node name="ar_track_alvar" pkg="ar_track_alvar" type="individualMarkers" respawn="false" output="screen" args="$(arg marker_size) $(arg max_new_marker_error) $(arg max_track_error) $(arg cam_image_topic) $(arg cam_info_topic) $(arg output_frame)"/>

</launch>